onnxruntime/contrib_ops/cpu/attnlstm/uni_dir_attn_lstm.cc

```diff
@@ -45,7 +45,7 @@ UniDirectionalAttnLstm<T>::UniDirectionalAttnLstm(AllocatorPtr allocator,
                                                   const ActivationFuncs::Entry& activation_func_g,
                                                   const ActivationFuncs::Entry& activation_func_h,
                                                   const float clip,
-                                                  onnxruntime::concurrency::ThreadPool& ttp)
+                                                  onnxruntime::concurrency::ThreadPool* ttp)
```

Comment:
The PR description said there's going to be a separate lstml thread pool. I don't see it in this iteration. Is that going to be in a separate PR?

Reply:
The separated thread pool is only needed for the UniDirectionalLstm class.